repo/fsrepo/fsrepo.go

```diff
@@ -578,6 +581,26 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 	return du, err
 }
 
+func (r *FSRepo) SwarmKey() ([]byte, error) {
```

Comment:
Word, it would be great if we could move the private key out of the config file in a similar fashion :+1: